.github/workflows/github-metadata.yml

```diff
@@ -0,0 +1,23 @@
+name: github_metadata
+on:
+  pull_request_target:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
+
+jobs:
+  check:
+    name: protect github metadata
```

Comment:
Bikeshedding: "Protect GitHub workflow changes"

Reply:
There's other things (like codeowners and so) that can live in `.github/`.  It's not just workflows.

Reply:
OK, fair enough. Calling it "meta"data is still a bit weird though. And our other "name:"s are capitalized.